packages/jest-snapshot/src/__tests__/SnapshotFile-test.js

```diff
@@ -99,4 +99,21 @@ describe('SnapshotFile', () => {
     expect(snapshotFile.get(SNAPSHOT)).toBe('"' + SNAPSHOT_VALUE + '"');
     snapshotFile.save();
   });
+
+  it('sorts snapshots by natural sort order', () => {
+    const fs = require('fs');
+    const snapshotFile = SnapshotFile.forFile(TEST_FILE);
+    ['test 2', 'test 11', 'test 1', 'test 10'].forEach(snapshot =>
+      snapshotFile.add(snapshot, SNAPSHOT_VALUE),
+    );
+    fs.writeFileSync.mockImplementation((path, content) => {
+      expect(content).toBe(
+        'exports[`test 1`] = `"bar"`;\n\n' +
+        'exports[`test 2`] = `"bar"`;\n\n' +
+        'exports[`test 10`] = `"bar"`;\n\n' +
+        'exports[`test 11`] = `"bar"`;\n',
+      );
+    });
+    snapshotFile.save();
```

Comment:
I wasn't fan of the expects inside the mocks either, so I've converted all the tests in this file to use this style with `.toBeCalledWith(...)` instead.

Also found out that there was a `fs.readFileSync` mock even though we don't use `readFileSync` for loading the snapshots anymore, so I removed the mock after verifying it's never called.